zip-cos7-ppc64le/meta.yaml

```diff
@@ -3,14 +3,14 @@ package:
   version: 3.0
 
 source:
-  - url: http://mirror.centos.org/altarch/7.8.2003/os/ppc64le/Packages/zip-3.0-11.el7.ppc64le.rpm
+  - url:  {{ centos7_url }}/Packages/zip-3.0-11.el7.ppc64le.rpm
```

Comment:
I think Packages should be removed here.

Reply:
Oops!